examples/plot_clip.py

```diff
@@ -0,0 +1,156 @@
+"""
+Clip Vector Data with GeoPandas
+==================================================================
+
+Learn how to clip point, line, or polygon geometries to the boundary
+of a polygon geometry using GeoPandas.
+
+"""
+
+###############################################################################
+# Clip Vector Data in Python Using GeoPandas
+# ---------------------------------------------
+#
+# .. note::
```

Comment:
I'm not sure you need a note like this at the top of the vignette. i'd remove these lines